repositories/src/main/scala/cromwell/pipeline/datastorage/dto/project/configuration/ProjectConfigurationResponse.scala

```diff
@@ -0,0 +1,10 @@
+package cromwell.pipeline.datastorage.dto.project.configuration
+
+import cromwell.pipeline.datastorage.dto.{ProjectFileConfiguration, ProjectId}
+import play.api.libs.json.{Json, OFormat}
+
+final case class ProjectConfigurationResponse(projectId: ProjectId, projectFileConfigurations: List[ProjectFileConfiguration])
```

Comment:
I don't like the idea of having the same `ProjectConfigurationResponse` and `ProjectConfigurationRequest`. 
In this case more appropriate way is to create special single entity for the Mongo.